### PR TITLE
[NCP-3449] showing the completion status of dns01challenge

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: en_US.json [hunk 1]
```diff
@@ -252,6 +252,11 @@
 	"certificate.dns01challenge.opt.desc": "This option allows us to automatically create a certificate supporting each of your hostnames after verifying your ownership of them.  To verify, you must modify your DNS server to add a special DNS TXT record for each hostname. Once the process is completed, you can remove those TXT records.",
 	"certificate.dns01challenge.radio.title": "DNS-01 challenge",
 	"certificate.dns01challenge.requesting": "Requesting",
+	"certificate.dns01challenge.steps.s1.title": "Initial verification",
+	"certificate.dns01challenge.steps.s2.title": "Extra Wait for DNS Propagation",
+	"certificate.dns01challenge.steps.s3.title": "Let’s Encrypt Confirmation",
+	"certificate.dns01challenge.steps.s4.title": "Certificate Upload",
+	"certificate.dns01challenge.steps.try.again.message": "Sorry, you are really unlucky today. Please try again at a later time.",
```
Comment (burtonquantil) [inline en_US.json:259]: I don't like the comment about being unlucky. Are there reasons the DNS challenge could not be completed?
How likely is the customer to see this error? How confident are you that trying again later will work?

Maybe we just simply say:

"Sorry, the DNS verification could not be completed. Please try again at a later time."

**Review by burtonquantil — CHANGES_REQUESTED** [review]
posted a comment and suggestion.

**Comment by chiraags** [comment]
@jungho73 For reviewing the Chinese translation, please add Jianghai or Bin for review.

File: en_US.json (hunk 1 above)
Comment (jungho73) [reply en_US.json:259]: I will modify it as you suggested.
It is unlikely that users will ever see this message. If the user saw it, there are various causes, so I think the sentence you suggested would be the best. Of course, we can check the cause through logs.

File: zh-Hans.json [hunk 2]
```diff
@@ -252,6 +252,11 @@
 	"certificate.dns01challenge.opt.desc": "此选项允许我们在验证您对主机名的所有权后自动创建支持您的每个主机名的证书。要进行验证，您必须修改您的 DNS 服务器，为每个主机名添加一个特殊的 DNS TXT 记录。该过程完成后，您可以删除这些 TXT 记录。",
 	"certificate.dns01challenge.radio.title": "DNS-01 挑战",
 	"certificate.dns01challenge.requesting": "请求",
+	"certificate.dns01challenge.steps.s1.title": "初步验证",
+	"certificate.dns01challenge.steps.s2.title": "额外等待 DNS 传播",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:256]: Please change it to: "等待DNS记录生效"

File: zh-Hans.json [hunk 3]
```diff
@@ -252,6 +252,11 @@
...
+	"certificate.dns01challenge.steps.s3.title": "让我们加密确认",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:257]: Please change it to: "CA验证DNS控制权"

File: zh-Hans.json [hunk 4]
```diff
@@ -252,6 +252,11 @@
...
+	"certificate.dns01challenge.steps.s4.title": "证书上传",
+	"certificate.dns01challenge.steps.try.again.message": "抱歉，你今天运气实在不好，请稍后再试。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:259]: Please change it to: "抱歉，DNS挑战失败，请稍后再试。"

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]
Some changes to Chinese texts are required.

**Comment by jungho73** [comment]
The sentences have been corrected.

**Review by burtonquantil — APPROVED** [review]
The English text is OK.

**Comment by jungho73** [comment]
@Jianghai-Quantil 
Could you check Chinese texts changed and approval?

File: en_US.json [hunk 5]
```diff
@@ -252,6 +252,11 @@
 	"certificate.dns01challenge.opt.desc": "This option allows us to automatically create a certificate supporting each of your hostnames after verifying your ownership of them.  To verify, you must modify your DNS server to add a special DNS TXT record for each hostname. Once the process is completed, you can remove those TXT records.",
 	"certificate.dns01challenge.radio.title": "DNS-01 challenge",
 	"certificate.dns01challenge.requesting": "Requesting",
+	"certificate.dns01challenge.steps.s1.title": "Initial verification",
```
Comment (Jianghai-Quantil) [inline en_US.json:255]: The initial letter of verification needs to be capitalized to be consistent with other titles here below.

**Review by Jianghai-Quantil — APPROVED** [review]
The Chinese texts look ok now. Posted a comment to the English text.

**Pull request merged**